### ISSUE TITLE
false positive, unused variable

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3572]
```abap
DATA name TYPE string.
PERFORM sdf IN PROGRAM (name).
```
